app-backend/migrations/src_migrations/main/scala/54.scala

```diff
@@ -0,0 +1,15 @@
+import slick.driver.PostgresDriver.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M54 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(54)(List(
+    sqlu"""
+    CREATE TABLE organization_features (
+      organization UUID NOT NULL UNIQUE,
+      feature_flags UUID[],
```

Comment:
That's a slick idea (no pun intended...oh wait, are we done with slick now?), but I would prefer an explicit DB and a thinner service layer. Not sure it matters a whole lot, but others may have stronger feelings/reasons for this.